vaadin-testbench-unit-shared/src/main/kotlin/com/vaadin/testbench/unit/internal/Shortcuts.kt

```diff
@@ -85,9 +81,13 @@ private class MockFilterJsonObject(val key: Key, val modifiers: Set<Key>) : JreJ
     companion object {
         private val mgenerateEventKeyFilter = ShortcutRegistration::class.java.getDeclaredMethod("generateEventKeyFilter", Key::class.java)
         private val mgenerateEventModifierFilter = ShortcutRegistration::class.java.getDeclaredMethod("generateEventModifierFilter", Collection::class.java)
+        private val chashableKey =  Class.forName(
+            "com.vaadin.flow.component.ShortcutRegistration\$HashableKey"
+        ).declaredConstructors[0]
```

Comment:
very last thing is we could have this:
```
 ShortcutRegistration::class.java
    .classLoader
    .loadClass("com.vaadin.flow.component.ShortcutRegistration\$HashableKey")
    .declaredConstructors[0]
```
Not sure if it is better or not, maybe a bit more Kotlin syntax.